addons/auth_signup/models/res_users.py

```diff
@@ -143,6 +143,15 @@ def action_reset_password(self):
             template.with_context(lang=user.lang).send_mail(user.id, force_send=True, raise_exception=True)
             _logger.info("Password reset email sent for user <%s> to <%s>", user.login, user.email)
 
+    @api.model
+    def web_dashboard_create_users(self, emails):
+        inactive_users = self.search(['|', ('login', 'in', emails), ('email', 'in', emails)]).filtered(lambda user: user.state == 'new')
```

Comment:
Why not put everything in the search domain using something like this : 
`self.search([('state', '=', 'new'), '|', ('login', 'in', emails), ('email', 'in', emails)])`

Reply:
@RomainLibert , 'State' is non stored compute field so we can not use it in the search.
so would like to do with log_ids like:
`self.search([('log_ids', '=', False), '|', ('login', 'in', emails), ('email', 'in', emails)])`

Reply:
Then we should define a search method for this field

Reply:
@RomainLibert , Instead of defining search I would like do directly check for log_ids because it's calculated based on log_ids.create_date. (log_ids is also in res_user)

Reply:
IMHO the search method is cleaner because it will clearly show that you are looking for a new user, the other benefit we get from using the search method is that if we ever change the way we compute the state we just have to update the search method and it will be ok.

Reply:
@RomainLibert  done changes